src/hw_isolation_record/entry.cpp

```diff
@@ -70,6 +70,16 @@ void Entry::delete_()
     // throws exception if not allowed
     hw_isolation::utils::isHwDeisolationAllowed(_bus);
 
+    // throws exception if the user tried deisolate the system
```

Comment:
- Note, The openpower-hw-isolation still allows the system isolated
  hardware entries to clear if the user tried through "DeleteAll".
> why the rule is different for delete and delete all?

Reply:
We need to allow the user in a certain situation to delete so, DeleteAll is acting like force invalidate all the records